hedgehog/src/Hedgehog/Internal/Check.hs

```diff
@@ -0,0 +1,65 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+
+module Hedgehog.Internal.Check
+  ( check
+  ) where
+
+import           Control.Monad.IO.Class (MonadIO(..))
+import           Hedgehog.Internal.Config
+import           Hedgehog.Internal.Prelude
+import           Hedgehog.Internal.Property
+import           Hedgehog.Internal.Region
+import           Hedgehog.Internal.Report
+import           Hedgehog.Internal.Runner (checkReport)
+import           Hedgehog.Range (Size)
+
+checkRegion ::
+     MonadIO m
+  => Region
+  -> UseColor
+  -> Maybe PropertyName
+  -> Size
+  -> Seed
+  -> Property
+  -> m (Report Result)
+checkRegion region color name size seed prop =
```

Comment:
You forgot to remove it from hedgehog/src/Hedgehog/Internal/Runner.hs

Reply:
I cannot move those because they are needed by other functions in the `Runner` module.